docs/content/dagster-cloud/managing-deployments/alerts/pagerduty.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: "PagerDuty & Dagster Cloud | Dagster Docs"
+description: "Create alerts that make PagerDuty incidents based on events in Dagster Cloud."
+
+platform_type: "cloud"
+feature: "alerting"
+integration: "pagerduty"
+---
+
+# PagerDuty & Dagster Cloud
+
+<ApplicableDagsterProduct />
+
+Dagster Cloud supports [sending alerts](/dagster-cloud/managing-deployments/alerts) to PagerDuty, which will automatically create a PagerDuty incident.
+
+---
+
+## PagerDuty + Dagster Cloud integration benefits
+
+- Empower your team with real-time incident response by integrating Dagster Cloud and PagerDuty
+- Notify on-call responders when alerts are triggered in Dagster Cloud
+- Dramatically reduce downtime for issues related to asset materialization, asset checks, job runs, schedules, sensors, and code locations
+- Customize the urgency and routing of PagerDuty incidents based on the content of the Dagster Cloud event payload
+
+---
+
+## How it works
+
+- Events that trigger an alert in Dagster Cloud will automatically create a PagerDuty incident
+- Incidents can be triggered for events related to runs, assets, schedules, sensors, and code locations
+- Users must resolve incidents in PagerDuty. Dagster Cloud doesn't currently automatically resolve incidents.
+
+### Requirements
+
+To integrate PagerDuty with Dagster Cloud, you'll need:
+
+- **Permissions in PagerDuty that allow you to manage services and integrations.** Refer to PagerDuty's [User roles](https://support.pagerduty.com/docs/user-roles) documentation for more information.
+- **The following in Dagster Cloud**:
+  - **A Pro plan**
```

Comment:
I think we could add a callout for this

Reply:
that this is enterprise only

Reply:
probably fine here, but I was thinking something at the top of the page. Not critical

Reply:
I could add it to the callout for Dagster Cloud at the top of the page. Something like "This guide is applicable to Dagster Cloud Pro plans."

Reply:
perfect